AirCasting/URLProviders/BaseURLProvider.swift

```diff
@@ -5,10 +5,12 @@ import Foundation
 
 protocol BaseURLProvider {
     var baseAppURL: URL { get set }
+    var sessionSynced: Bool { get set }
```

Comment:
why should an object designed for providing us with URL have such a flag also? :) Not a good place for that

Reply:
Yeah, you are right :/